new/LIT5901DersanaGabrielNaglon.xml

```diff
@@ -0,0 +1,83 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5901DersanaGabrielNaglon" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+              <title xml:lang="en" corresp="#t1">Homily on construction of a church of Archangel Gabriel at Naglon (Naqlūn)</title>
```

Comment:
the construction
Is this the title to be printed? Then the other ones whould have t1, and this one xml:id="t2" type="main"

Reply:
changed.